### ISSUE TITLE
Increment setting using `conventional-changelog- *` doesn't work.

### ISSUE THREAD

**Issue by 1natsu172** [webpro/release-it#260]
Hi @webpro , I faced a new problem. 

In a project using "conventional-changelog-atom", I want to automate Increment, so I set it up. But it doesn't work. This is happening in both the local & CI.

**.release-it.json**

```json
{
  "increment": "conventional:atom",
}
```

## Logs

`--debug`&`--dry-run`

```
$ git describe --tags --abbrev=0
v1.2.8
  release-it:shell { command: 'git describe --tags --abbrev=0',
  release-it:shell   options: { isReadOnly: true },
  release-it:shell   code: 0,
  release-it:shell   stdout: 'v1.2.8',
  release-it:shell   stderr: '' } +34ms
WARNING Unable to find an existing Git tag, or package.json#version. No or invalid version provided.
error Command failed with signal "SIGINT".
```

An error occur in the process of parsing the latest version of Git. I don't know why .... 🤷‍♂️ 

But If I delete the `"Increment"` field, this error will disappear. Off cause, I installed  `conventional-changelog-atom": "^0.2.3"`. "release-it" version is `"^7.2.0"`.

---

`release-it/lib/version.js`

https://github.com/webpro/release-it/blob/1666e8309498adf3476c182194458a29db88931c/lib/version.js#L17

I'm curious that the way of setting this line is different from the definition of "conventional-recommended-bump" somewhat, but this may not be a problem. 

* https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-recommended-bump

Sorry I have not clearly followed the bug. But there seems to be a problem something around Bump.

**Comment by webpro** [comment]
Thanks for the bug report, @1natsu172. I forgot to remove a piece of code when starting to support all conventional-changelog presets. Fixed in [v7.2.1](https://github.com/webpro/release-it/releases/tag/7.2.1).

Btw, you don't need to install the preset yourself, it should come with release-it.

**Comment by 1natsu172** [comment]
Thanks very fast fixing bug, @webpro . I updated to v7.2.1 and run . It works well with `conventional-changelog-atom` style!

> Btw, you don't need to install the preset yourself, it should come with release-it.

wow, certainly `conventional-changelog` is included in dependencies… thanks for teaching me. 😃
